core/best-practices/index.md

```diff
@@ -122,7 +122,7 @@ that have unreasonable durations
 ### Pupil Detection and Blinks
 It is worth noting that poor pupil detection in general can lead to false negatives. In such instances, adjusting the 
 thresholds can make it easier to detect blinks, but also increases the chance of false positives. It is worth taking the 
-time to ensure an optimal setup with regards to [eye camera positioning](/#_3-check-pupil-detection) and 
+time to ensure an optimal setup with regards to [eye camera positioning](/getting-started/index.md#3-check-pupil-detection) and 
```

Comment:
@weePLs I think the link is wrong. It should be 

/getting-started/#_3-check-pupil-detection

instead of getting-started/index.md#3-check-pupil-detection

Can you fix and confirm it works?

Reply:
Hmm, I don't know if I did something wrong, but that URL didn't work for me.

![image](https://github.com/user-attachments/assets/0be36e89-64aa-44af-a0ae-3d8f4b5901e1)

Reply:
Oh, nevermind the previous post. Yes, it works!